Sources/EdgeNetworkHandlers/RequestBuilder.swift

```diff
@@ -97,6 +97,10 @@ class RequestBuilder {
                 eventData[Constants.JsonKeys.XDM] = xdm
             }
 
+            if let query = eventData[Constants.JsonKeys.QUERY] as? [String: Any] {
```

Comment:
Let's remove this as it appears to just set the same value back to `eventData`. I'm seeing the tests pass after removing this statement.